astropy/nddata/utils.py

```diff
@@ -191,7 +191,7 @@ def extract_array(array_large, shape, position, mode='partial',
         array will be filled with ``fill_value``.  In ``'trim'`` mode
         only the overlapping elements are returned, thus the resulting
         small array may be smaller than the requested ``shape``.
-    fill_value : number, optional
+    fill_value : float or int, optional
```

Comment:
Why did change change "number" to "float or int" here, but kept "number" on L176?

Reply:
I'll revert. That change presumably predated adding "number" to the glossary, which happened ... a long thread ago.